include/kernel.h

```diff
@@ -1708,9 +1707,8 @@ void k_disable_sys_clock_always_on(void);
  *    @rst
  *    While this function returns time in milliseconds, it does
  *    not mean it has millisecond resolution. The actual resolution depends on
- *    :option:`CONFIG_SYS_CLOCK_TICKS_PER_SEC` config option, and with the
- *    default setting of 100, system time is updated in increments of 10ms.
- *    @endrst
+ *    :option:`CONFIG_SYS_CLOCK_TICKS_PER_SEC` config option
+ *    @endrststar
```

Comment:
doxygen brackets for embedded restructured text are now simply ``@rst`` ... ``@endrst``  (we got rid of the @endrststar alias to simplify things).  it was correct as ``@endrst`` in the previous version of this file.